src/core/manage.py

```diff
@@ -355,6 +365,26 @@ def upload_to(self, filename):
             app.logger.critical("Upload failed!")
             abort()
 
+    def download_update_file(self, filename, url):
+        import requests
+        import gzip
+
+        try:
+          app.logger.debug("start download of {0}".format(url))
+          r = requests.get(url, allow_redirects=True)
+          content_type = r.headers.get('Content-Type')
+          content_enc = r.headers.get('Content-Encoding')
+          if content_type == 'application/x-gzip' or content_enc == 'x-gzip':
+            content = gzip.decompress(r.content)
+          else:
+            if content_type != 'application/xml':
+              app.logger.debug("{0} does not point to an XML content-typed file".format(url))
+            content = r.content
+          open(filename, 'wb').write(content)
```

Comment:
Missing context
Is writing as bytes necessary?

Reply:
decission should match the "old" behaviour which uses 'wb'
Could be reevaluated to swich both